python_modules/libraries/dagster-polars/dagster_polars_tests/test_polars_delta.py

```diff
@@ -22,63 +19,9 @@
 from dagster_polars import PolarsDeltaIOManager
 from dagster_polars.io_managers.delta import DeltaWriteMode
 from deltalake import DeltaTable
-from hypothesis import given, settings
-from polars.testing.parametric import dataframes
 
 from dagster_polars_tests.utils import get_saved_path
 
-# TODO: remove pl.Time once it's supported
-# TODO: remove pl.Duration pl.Duration once it's supported
-# https://github.com/pola-rs/polars/issues/9631
-# TODO: remove UInt types once they are supported
-#  https://github.com/pola-rs/polars/issues/9627
-
-
-@pytest.mark.flaky(reruns=5)
-@given(
-    df=dataframes(
-        excluded_dtypes=[
-            pl.Categorical,
-            pl.Duration,
-            pl.Time,
-            pl.UInt8,
-            pl.UInt16,
-            pl.UInt32,
-            pl.UInt64,
-            pl.Datetime("ns", None),
-            pl.Decimal,
```

Comment:
i spent an hour trying to update this exclude list against new polars `0.20.26` and was unsuccessful

Reply:
They were in the exclude list because they are unsupported types in delta protocol